fdbserver/RestoreLoader.actor.cpp

```diff
@@ -478,8 +478,8 @@ void _parseSerializedMutation(std::map<LoadingParam, VersionedMutationsMap>::ite
 			    .detail("CommitVersion", commitVersion)
 			    .detail("ParsedMutation", mutation.toString());
 			kvOps[commitVersion].push_back_deep(kvOps[commitVersion].arena(), mutation);
-			// Sampling (FASTRESTORE_SAMPLING_RATE * 100 %) data
-			if (deterministicRandom()->randomInt(0, 10000) < 10000 * SERVER_KNOBS->FASTRESTORE_SAMPLING_RATE) {
+			// Sampling (FASTRESTORE_SAMPLING_PERCENT * 100 %) data
+			if (deterministicRandom()->randomInt(0, 10000) < 10000 * SERVER_KNOBS->FASTRESTORE_SAMPLING_PERCENT) {
```

Comment:
this should be `deterministicRandom()->random01() * 100 < SERVER_KNOBS->FASTRESTORE_SAMPLING_PERCENT` or
`deterministicRandom()->randomInt(0, 10000) < 100 * SERVER_KNOBS->FASTRESTORE_SAMPLING_PERCENT`, since `0 <= FASTRESTORE_SAMPLING_PERCENT <= 100`.

Reply:
Ah, I defined `0 < FASTRESTORE_SAMPLING_PERCENT  < 1`. 
I won't mind changing to what you suggested, which don't have a limitation on the precision.